packages/ember-testing/lib/helpers/pause_test.js

```diff
@@ -1,3 +1,17 @@
+/*
```

Comment:
Can you move this down to just above the `pauseTest` function below, and update the comment open to be `/**`?